ls1intum/Artemis
# `Development`: Replace deprecated method call

### Checklist
#### General

### Motivation and Context
Replaces a deprecated method call

### Description
`copyPattern(String)` is deprecated in favour of `copyPatterns(String...)`

### Steps for Testing
Create a new PE with static code analysis enabled. Check that issues are correctly reported back to Artemis.

### Review Progress
#### Code Review
#### Manual Tests

### Screenshots

copy patterns in the newly created build plan:
![grafik](https://github.com/ls1intum/Artemis/assets/26540346/e0e7ab74-ef1b-4d58-bcdc-293f48754510)
